shared/src/simulation/events/patient-category-transfer-to-hospital-finished.ts

```diff
@@ -0,0 +1,38 @@
+import { IsBoolean } from 'class-validator';
+import { getCreate } from '../../models/utils';
+import { IsLiteralUnion, IsValue } from '../../utils/validators';
+import {
+    AlivePatientTriageCategory,
+    alivePatientTriageCategoryAllowedValues,
+} from '../utils/alive-patient-triage-category';
+import type { SimulationEvent } from './simulation-event';
+
+export class PatientCategoryTransferToHospitalFinishedEvent
+    implements SimulationEvent
+{
+    @IsValue('patientCategoryTransferToHospitalFinishedEvent')
+    readonly type = 'patientCategoryTransferToHospitalFinishedEvent';
+
+    @IsLiteralUnion(alivePatientTriageCategoryAllowedValues)
+    readonly patientCategory: AlivePatientTriageCategory;
```

Comment:
Is there any real reason why this should not be any patient category? What about `blue` or `white`? They are mostly alive, aren't they?

Reply:
I think a name like RelevantForHospitalTransportPatientCategory makes it more clear what it is. And all other patients than SK1-3 will not be relevant to the Transfer Behavior so I let them out

Reply:
It would have been less work *and* more robust if left in. What if a patient dies while the transport is being prepared? What if the participants decide to transport a blue patient regardless?

Reply:
In case 1) The patient must not be transported to not waste important capacity, In case 2) It is not relevant to the behavior and therefore this event should not be sent. 
The Transfer patients to Hospital behavior only knows 3 categorie (we could change this but that is how its currently planned) and therefor this data type should represent exactly those 3 categories

Reply:
done